Python/bltinmodule.c

```diff
@@ -3163,7 +3162,7 @@ static struct PyModuleDef builtinsmodule = {
     NULL
 };
 
-
+extern PyTypeObject PyRegion_Type;
```

Comment:
Perhaps separate with a new line here.
```suggestion
extern PyTypeObject PyRegion_Type;

```